### PR TITLE
Added value check to init method

### PR DESCRIPTION
**Pull request by temurchichua**
Added the value check to the init method.

###  Summary

I've set the env. variable for the Slack Event Token and used it to create Adapter as following:

```python
slack_event_adapter = SlackEventAdapter(os.environ.get("SLACK_EVENTS_TOKEN"), "/slack/events", app)
```

I made a typo while setting the variable, thus I was getting "Your URL didn't respond with the value of the challenge parameter." on the Event Subscription page with the following error in my terminal:

```bash
Traceback (most recent call last):
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\slackeventsapi\server.py", line 100, in event
    if req_signature is None or not self.verify_signature(req_timestamp, req_signature):
  File "D:\PycharmProjects\Slackbot\flipper\venv\lib\site-packages\slackeventsapi\server.py", line 60, in verify_signature
    str.encode(self.signing_secret),
TypeError: descriptor 'encode' for 'str' objects doesn't apply to a 'NoneType' object
```

I took some time to look through the source code and see what went wrong. I think value checking could come in handy for other developers to easily find the error in their project.

### Requirements (place an `x` in each `[ ]`)

* [X] I've read and understood the [Contributing guidelines](https://github.com/slackapi/python-slack-events-api/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [X] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Comment by seratch** [comment]
Hi @temurchichua, thanks for taking the time to make this pull request! We are happy to merge this improvement but I have a few comments. Can you check them when you have a chance?

File: .gitignore [hunk 1]
```diff
@@ -15,4 +15,5 @@ tests/.cache
 .vscode/
 .eggs/
 build/
-.python-version
\ No newline at end of file
+.python-version
+venv/
```
Comment (seratch) [inline .gitignore:19]: I know you use `venv/` only but many people may use `.venv`. Can you add the following ones as well?
```suggestion
venv/
.venv/
.env/
```

File: slackeventsapi/__init__.py [hunk 2]
```diff
@@ -7,7 +7,11 @@ class SlackEventAdapter(BaseEventEmitter):
     # If no endpoint is provided, default to listening on '/slack/events'
     def __init__(self, signing_secret, endpoint="/slack/events", server=None, **kwargs):
         BaseEventEmitter.__init__(self)
-        self.signing_secret = signing_secret
+        if signing_secret:
+            self.signing_secret = signing_secret
+        else:
+            raise ValueError(f'Got {type(signing_secret)} instead of the Slack signing secret token. Check the first '
+                             f'parameter.')
```
Comment (seratch) [inline slackeventsapi/__init__.py:14]: How about doing like this?
```suggestion
        if signing_secret is None:
            message = "signing_secret is required but you passed None in the first argument"
            raise ValueError(message)
        self.signing_secret = signing_secret
```

**Review by seratch — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (temurchichua) [reply .gitignore:19]: That sounds right to me.

File: slackeventsapi/__init__.py (hunk 2 above)
Comment (temurchichua) [reply slackeventsapi/__init__.py:14]: Yes, that looks more Pythonic.

**Comment by temurchichua** [comment]
@seratch thanks for reviewing the pull request.
Good points. I've included changes in the commit.

**Review by seratch — APPROVED** [review]
LGTM

**Comment by seratch** [comment]
@temurchichua Thanks for the contribution!

**Pull request merged**